src/examples/example.scala

```diff
@@ -10,7 +10,7 @@ var dynamicLang = "es"
 @main
 def run(lang: String): Unit =
    def number(n: Int): Messages[MyLangs] = n match
-      case 1 => en"one" & fr"un" & de"ein" & es"uno"
+      case 1 => en"one" & fr"un" & de"ein" & es"uno" // & fr"uno" // KO
```

Comment:
So, I thought this was working (i.e. not compiling) with the old `NotGiven` test...